.github/workflows/build/action.yaml

```diff
@@ -55,25 +55,28 @@ runs:
         branch: ${{ github.head_ref }}
         ssh: true
 
-    - name: Dry run tarballs
```

Comment:
We could still run this on GH runners, but conditioning with [if ](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context)runner.name[ != something](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context).

Or do you have a plan on how to run this on the custom runners in the (near) future?

Reply:
It will be super hard to manage cache between self hosted runners and github runners. I will look into fixing this. I know what the issue is just have to see what are all the possibilities of fixing it

Reply:
I don't think that's necessary. We can let GH rebuild the cache on master every now and then (e.g. once per week), that should be enough IMHO.